packages/notifications/src/elements/toaster/styled.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
+import { hideVisually } from 'polished';
+import { TransitionGroup } from 'react-transition-group';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { ToastPlacement } from './reducer';
+
+export const TRANSITION_CLASS = 'garden-toast-transition';
+
+export const StyledFadeInTransition = styled.div<{ isHidden: boolean; placement: ToastPlacement }>`
+  transition: opacity 400ms ease-in 300ms;
```

Comment:
It would be good to factor `400ms` into a constant and then update all following values based on this constant. This way, we clearly understand which timing depends upon another.

Reply:
👍 Updated to a shared constant.